editor/editor_interface.h

```diff
@@ -54,12 +54,25 @@ class Texture2D;
 class Theme;
 class VBoxContainer;
 class Window;
+class SceneTreeDialog;
+class PropertySelector;
 
 class EditorInterface : public Object {
 	GDCLASS(EditorInterface, Object);
 
 	static EditorInterface *singleton;
 
+	// Editor dialogs.
+
+	SceneTreeDialog *dialog_scene_tree = nullptr;
+	PropertySelector *dialog_property_selector = nullptr;
```

Comment:
```suggestion
	SceneTreeDialog *scene_tree_dialog = nullptr;
	PropertySelector *property_selector_dialog = nullptr;
```

There is no reason for a reverse order. Can also rename them after the methods (I'll explain below).